anago

```diff
@@ -742,6 +743,23 @@ get_build_candidate () {
                                 $PARENT_BRANCH \
    || return 1
 
+  # When creating alphas, make sure that X.X.0 has been created first or warn.
```

Comment:
I think the important part is blocking `branchff` if you've passed the point of no return. The warning here is just saying, "You're about to pass the point of no return. Are you sure you want to do this?"